src/file-engine/src/engine.rs

```diff
@@ -116,6 +119,20 @@ impl RegionEngine for FileRegionEngine {
         }
     }
 
+    async fn set_writable_gracefully(
+        &self,
+        region_id: RegionId,
+        _entry_id: Option<entry::Id>,
+    ) -> Result<SetWritableResponse, BoxedError> {
+        let exists = self.inner.get_region(region_id).await.is_some();
+
+        if exists {
+            Ok(SetWritableResponse::Result(Ok(())))
```

Comment:
File engine region is essentially not writable, I think it's better to just return an error early.

Reply:
Here is a tradeoff: if users attempt to move the file table, the operation will fail. Maybe, we should add a region migration process for the file engine later on.

Reply:
No need. Actually I think we can simply `unimplemented!()` here.

Reply:
Well, panic is not good, `return UnsupportedSnafu` better.